homeassistant/components/onvif/parsers.py

```diff
@@ -331,16 +344,15 @@ async def async_parse_last_reboot(uid: str, msg) -> Event:
 
     Topic: tns1:Monitoring/OperatingTime/LastReboot
     """
+    date_time = datetime_or_zero(msg.Message._value_1.Data.SimpleItem[0].Value)
```

Comment:
Should this be within the try... except block? What about the other similar lines?

Reply:
Done 8cc2b636